org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManager.java

```diff
@@ -207,7 +212,65 @@ private Set<IPath> removeImportedConfigurations(Set<IPath> configurations, IProj
 			.collect(Collectors.toSet());
 	}
 
-	public void importProjects(IProgressMonitor monitor) {
+	public void importProjects(IProgressMonitor monitor, String... buildFileUris) {
```

Comment:
To avoid NoSuchMethodError when `importProjects(IProgressMonitor)` is invoked somewhere. `String...` makes the second argument optional. If its unavailable, goes to the automatic mode.

Reply:
Hmm, it's invalid now. If we want to resolve the toAdd/toRemove/toUpdate sets at the client side and pass then to the server, then I guess I need to add a new method for that purpose.